### PR TITLE
v0.0.1

### PR DESCRIPTION
**Pull request by arpitkarsolia**
List of stable & validated micro-services for Revere Snowfox-

Applanix LV V5 GPS v0.0.10
Velodyne HDL32C Lidar v0.0.13
OpenCV Camera v0.0.10
x264 Encoder - master
OpenDLV Snowfox CAN
Sony PS3 Controller v0.0.10
OpenDLV Vehicle Viewer v0.0.53

### CODE REVIEW

File: snowfox.yml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        command: "nice -n 0 /opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=110"
+
+######################################################################################
+#
+# Microservices for SNOWFOX (Volvo XC90).
+#
+######################################################################################
+   
+    # RTK Corrections for Applanix (needs update)
+#    dlv-ntrip:
```
Comment (chrberger) [inline snowfox.yml]: We did not test this, right?

File: snowfox.yml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    # Velodyne HDL32 Lidar (10.42.42.70)
+    dev-lidar-hdl32e:
+        container_name: dev-lidar-hdl32e
+        image: chalmersrevere/opendlv-device-lidar-hdl32e-multi:v0.0.13
+        restart: always
+        network_mode: "host"
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n -7 /usr/bin/opendlv-device-lidar-hdl32e --hdl32e_ip=0.0.0.0 --hdl32e_port=2368 --cid=110"
```
Comment (chrberger) [inline snowfox.yml]: Shouldn't we add `--gpstime` here?

File: snowfox.yml (hunk 1 above)
Comment (arpitkarsolia) [reply snowfox.yml]: yes, we did not. It is not being used right now.

File: snowfox.yml (hunk 2 above)
Comment (arpitkarsolia) [reply snowfox.yml]: done

**Pull request merged**